README.md

```diff
@@ -16,15 +16,32 @@ For more details please visit the [documentation](./DOCUMENTATION.md) page.
 
 ## Build it yourself
 
+Download the source code
 ```
-$ git clone https://github.com/tchx84/Flatseal.git
-$ cd Flatseal
-$ flatpak-builder --force-clean --repo=repo build com.github.tchx84.Flatseal.json
-$ flatpak build-bundle repo flatseal.flatpak com.github.tchx84.Flatseal
-$ flatpak install flatseal.flatpak
+git clone https://github.com/tchx84/Flatseal.git
+cd Flatseal
 ```
 
-Or just use [Builder](https://flathub.org/apps/details/org.gnome.Builder)
+
+**Build it with Flatpak**
+
+```
+flatpak install org.gnome.{Platform,Sdk}//41
+flatpak-builder --user --install build com.github.tchx84.Flatseal.json
+flatpak run --branch=master com.github.tchx84.Flatseal
+```
+
+
+**Build it with Builder**
+
+```
+flatpak install org.gnome.Builder
+flatpak run org.gnome.Builder
```

Comment:
I think these instructions are out of scope for this README, please keep only the "build with flatpak" steps.

Reply:
Ok. Fixed in 57f0edf